### DISCUSSION TITLE
Are you self hosting splitpro?

### DISCUSSION THREAD

**Discussion by KMKoushik** [oss-apps/split-pro#93] [Announcements]
Hello all,

If you're self hosting splitpro, please let me know.

I'm making lot's of changes to the app and I can send updates when I release new version so you are aware of what's happening.

You can also send an email to hello@ossapps.dev

**Comment by chiel1112** [comment]
I am selfhosting it!

**Reply by KMKoushik** [reply]
hello, can you send an email to me so i can add you to the list!

**Reply by chiel1112** [reply]
Done!

**Comment by delta0815** [comment]
I am selfhosting it as well... It runs un Unraid. You might want to consider to build an Unraid-package and maintain it officially. It would probably bring you some more users :-)

**Comment by gravelfreeman** [comment]
Got it to work on k8s with Truecharts app-template featuring Traefik reverse proxy and cnpg backup and restore! Here's my [config](https://github.com/gravelfreeman/talos/blob/main/clusters/main/kubernetes/apps/splitpro/app/helm-release.yaml).

Really nice application and possible replacement for Splitwise in the future.

Here's what I came up with as missing and important for me to completely and effectively replace Splitwise.

In order of priority (in my opinion):

1. OIDC support for Authelia #148
2. Reccuring expenses #160
2. Categories and Tags system #161
3. Receipts scanning #162
4. Statistics #100

Why OIDC priority #1? I almost gave up trying to figure out next-auth until I understood that it comes with the image and that it's plug and play. In the meantime, I guess it would attract more people to explain that and also that the URL must be your public URL of SplitPro.

Good job and good luck with the project. It's a shame I can't code, I would've tried to help.

**Comment by hrmtc** [comment]
I'm selfhosting!
Thanks for your work!

**Comment by Silther** [comment]
I am selfhosting it too.
running on docker compose behind a cloudflare tunnel and with proton mail as smtp mail server

**Comment by masterpi007** [comment]
I am self-hosting with my customized version that fixed some bugs, but still prioritize the official repo
